Annotation/alternative_golden_set.json

```diff
@@ -810,7 +822,7 @@
                     }
                 ],
                 "type": "alternative",
-                "clause": "",
+                "clause": "whether during this thirty or forty year period of the low-fat dogma, whether breastfeeding rates went up or down",

```

Comment:
I think this is two nested EC? or coordinated? because its debating if its thirty or forty year and then debating breastfeeding rates

Reply:
I think the whether is not linked to the "or" in "thirty or forty" because it's a complete nominal phrase "this thirty of forty year *period*". So I guess I think it's just one alternative being wondered about: whether breastfeeding rates went up or down during this period of the low-fat dogma